cli/cmd/user/add.go

```diff
@@ -0,0 +1,78 @@
+package user
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func AddCmd(cfg *config.Config) *cobra.Command {
+	var orgName string
+	var projectName string
+	var email string
+	var role string
+
+	addCmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if orgName == "" {
+				orgName = cfg.Org
+			}
```

Comment:
Put `cfg.Org` in the flag default instead, then the user will see it in the help menu:
```go
addCmd.Flags().StringVar(&orgName, "org", cfg.Org, "Organization")
```